account_reconcile_oca/views/account_bank_statement_line.xml

```diff
@@ -325,11 +326,12 @@
         <field
             name="context"
         >{'default_journal_id': active_id, 'view_ref': 'account_reconcile_oca.bank_statement_line_form_reconcile_view'}</field>
-        <field name="view_mode">tree</field>
+        <field name="view_mode">tree,kanban</field>
         <field
             name="view_ids"
             eval="[(5, 0, 0),
-            (0, 0, {'view_mode': 'tree', 'view_id': ref('bank_statement_line_reconcile_view')})]"
+            (0, 0, {'view_mode': 'tree', 'view_id': ref('account_statement_base.account_bank_statement_line_tree')}),
```

Comment:
Change the order, first the kanban please :wink:

Reply:
I just sent a fix.